### PR TITLE
Doc: FAQ changes for `retain VS VIP over AKO reboot`

### PR DESCRIPTION
**Pull request by swathinsankaran**
This PR addresses the document changes for `retain VS VIP over AKO reboot`.

### CODE REVIEW

**Review by sudswasavi — APPROVED** [review]

File: docs/faq.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ After all relevant objects gets deleted from Avi, the reason is changed to objDe
 
 To re-create the objects in Avi, the configmap has to be edited to set deleteConfig: "false". After this, the condition in ako statefulset of type ako.vmware.com/ObjectDeletionInProgress is deleted automatically.
 
+**Note**: To cleanup the shared VS VIP that is retained, the user has to:
```
Comment (akshayhavile) [inline docs/faq.md:33]: should it be part of section "#### How is the Shared VS lifecycle controlled" with more elaboration of shared vs vip behaviour change?

File: docs/faq.md [hunk 2]
```diff
@@ -30,6 +30,9 @@ After all relevant objects gets deleted from Avi, the reason is changed to objDe
...
+1. Manually delete it from the controller UI or shell.
+2. Toggle the `enableEVH` flag in configmap and reboot the AKO.
```
Comment (akshayhavile) [inline docs/faq.md:35]: I did not get relevance of `enableEVH` flag to clean up shared vs vip

File: docs/faq.md (hunk 1 above)
Comment (swathinsankaran) [reply docs/faq.md]: I have moved it to the section `How is the Shared VS lifecycle controlled`.

File: docs/faq.md (hunk 2 above)
Comment (swathinsankaran) [reply docs/faq.md]: I have removed it as the flag `enableEVH` usage, in this case, is not the right way. Thanks.

**Review by akshayhavile — APPROVED** [review]

**Comment by swathinsankaran** [comment]
build ako

**Review by sudswasavi — APPROVED** [review]

**Pull request merged**